### PR TITLE
Branch11

### PR DESCRIPTION
**Pull request by MadhuBeesetty**

### CODE REVIEW

File: solutions/11.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const divisible = (a,b)=>{
+  if(a%b==0){
```
Comment (bramakrishna16) [inline solutions/11.js]: Please give appropriate names!!

File: solutions/11.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const divisible = (num1,num2)=>{
+  if(num1%num2==0){
```
Comment (bramakrishna16) [inline solutions/11.js:2]: Is it divisible by 7?

File: solutions/11.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const divisible = (num1,num2)=>{
```
Comment (allopez7) [inline solutions/11.js:1]: Only supposed to have one input number or parameter.

Comment (MadhuBeesetty) [reply solutions/11.js:1]: i took second parameter because to make my programme dynamic, so that i can take any number, not only 7.

**Pull request merged**